cdhweb/blog/exodus.py

```diff
@@ -57,4 +59,8 @@ def blog_exodus():
         # transfer attachments
         exodize_attachments(post, post_page)
 
-        # NOTE no tags to migrate
+        # exodize keywords as tags
+        for kw in post.keywords.all():
+            logging.debug("exodizing blog tag %s" % kw.keyword.title)
+            tag = Tag.objects.get_or_create(name=kw.keyword.title)[0]
+            BlogPostTag.objects.create(content_object=post_page, tag=tag)
```

Comment:
I'm surprised you have to do it this way — if it works the way taggit normally does, you ought to be able to just use `post.tags.add` with the list. (Not that you need to change it for this, since it's a one-time thing and clearly it works; just sharing as an FYI)

Reply:
again, just doing it for logging purposes. probably more verbosity than we need, but speed isn't really a concern, and I like to read everything!